R/install-git.R

```diff
@@ -146,9 +146,14 @@ remote_package_name.git2r_remote <- function(remote, ...) {
   description_path <- paste0(collapse = "/", c(remote$subdir, "DESCRIPTION"))
 
   if (grepl("^https?://", remote$url)) {
+    # assumes GitHub-style "<repo>/raw/<ref>/<path>" url
     url <- build_url(sub("\\.git$", "", remote$url), "raw", remote_sha(remote, ...), description_path)
-    download(tmp, url)
-    read_dcf(tmp)$Package
+    tryCatch({
+      download(tmp, url)
+      read_dcf(tmp)$Package
+    }, error = function(e) {
+      NA_character_      
```

Comment:
```suggestion
      return(NA_character_)
```

Yes I know that will just return 😄 But let's make it clear~

Reply:
In general we don't put explicit returns except for early returns. See https://style.tidyverse.org/functions.html#return for details.